imagine/priors/basic_priors.py

```diff
@@ -77,7 +77,7 @@ class GaussianPrior(ScipyPrior):
     def __init__(self, mu=0.0, sigma=1.0, xmin=None, xmax=None, unit=None,
```

Comment:
Also here I think we should get rid of defaults

Reply:
I would also change the order to make `xmin` and `xmax` come first, as it is with other priors.
Therefore, extra information is given after the usual information.

Reply:
Indeed, defaults here are bad. I change the order as well.

Reply:
On the other hand, now it is possible to set up a `GaussianPrior` without specifying `xmin` and `xmax`... so, I think I will keep the order for now